airflow/contrib/auth/backends/ldap_auth.py

```diff
@@ -157,8 +180,11 @@ def login(self, request):
         session.close()
 
         return redirect(request.args.get("next") or url_for("admin.index"))
-    except AuthenticationError:
-        flash("Incorrect login details")
+    except (LdapException, AuthenticationError) as e:
+        if e:
+            flash(e, "error")
```

Comment:
Looks like you'll always call this line if there is an exception rather than line 187, even though line 187 looks like it would match the old behavior more closely.  Maybe this is the source of the error.